packages/vscode-dependencies-validation/test/vscodeMocks.ts

```diff
@@ -0,0 +1,189 @@
+import { NPM_DEPENDENCY_ISSUES_CODE } from "../src/constants";
+import type {
+  Diagnostic,
+  DiagnosticCollection,
+  Uri,
+  CancellationToken,
+  CodeActionContext,
+  Event,
+  Position,
+  Range,
+  TextDocument,
+  TextLine,
+  CodeActionKind,
+} from "vscode";
+import { VscodeOutputChannel } from "../src/vscodeTypes";
+
+export const outputChannel: VscodeOutputChannel = {
+  append(value: string) {},
```

Comment:
if you add console.logs here we may get some useful tracing during tests.
or maybe it would make the test output ugly... worth a shot